doc/decisions/decision_process.md

```diff
@@ -129,23 +180,31 @@ These decision PRs are also merged for documentation purposes.
 - The template makes sure important points are not forgotten.
 - Every decision is by design in its own file with its own git history.
 - PRs allow to better support the constraint that everything must be within Elektra's repository (also rejected PRs).
+- "Decision", "Rationale" and "Implications" are filled out later to keep the discussion unbiased
 - PRs allow to suggest changes and review individual sentences of the decision.
 - Several "Related Decisions" are very important even if everyone agrees on one solution.
   They allow reviewers and future readers of the decision to understand which options were considered and why they were rejected.
+- The decision process is focused around the decision text (and not forum-like discussions), so that:
+  - The resulting text is understandable without reading any discussions.
+  - There is a common understanding after only reading the decision text.
+  - To avoid any gaps of reading discussions and the decision.
 
 ## Implications
 
 - Proposal issues are obsolete.
 - The decision process creates at least:
   - two chances to comment decisions, and
   - two commits in the git history.
+- It might be a barrier for newcomers to write a decision.
+  This is considered to be okay, as topics that need a decision are not the topics for newcomers.
```

Comment:
I guess you added that because of my recent remarks... But it doesn't address my concerns.

Here "newcomers" should refer to "people writing their first decision" not necessarily "people contributing to Elektra the first time". However, there is a lot of overlap. Many of the recent big changes that require decisions (change-tracking, new mounting, new kdb tool, new spec) are being done by people new to Elektra.

I wasn't actually concerned with the extra effort of writing a decision. Good documentation should be encouraged. That takes time, no way around that. My issue was that decisions are such a dry and theoretical process. _That_ can be very off-putting, if you had an idea and wanted to write code.

Reply:
Also the long and tedious process makes _me_ want to suggest fewer ideas, because I don't want to spend time on decisions.

Reply:
It is on purpose that less ideas get thrown around but instead we focus on decisions, which have much higher probability of actually getting done.